app/app.js

```diff
@@ -23,23 +23,9 @@ import App from 'containers/App';
 // Import Language Provider
 import LanguageProvider from 'containers/LanguageProvider';
 
-// Load the favicon, the manifest.json file and the .htaccess file
-/* eslint-disable import/no-webpack-loader-syntax */
+// Load the favicon and the .htaccess file
```

Comment:
Why'd we remove the multi-sized icons?

Reply:
We didn't. We keep only the largest icon (512*512) and, on build, webpack-pwa-manifest generates all of the other icons for us. They're automatically added in the build folder and linked to in the generated manifest.json.

Cf line 70 of webpack.prod.babel.js to see what I mean.

Reply:
Got it, I messed with it a bit more and I agree now with why its in this PR. Disregard